abstract.txt

```diff
@@ -0,0 +1,23 @@
+Open data formats and software for gamma-ray astronomy
+======================================================
+
+Authors: tbd
+
+In gamma-ray astronomy, a lot of proprietary data formats and software exist
+that were developed for one specific mission or experiment. Especially in
+ground-based gamma-ray astronomy, imaging atmospheric Cherenkov telescope (IACT)
+data and software until now was private to the collaborations operating the
```

Comment:
Maybe?  -> data and software has been private to the collaborations operating the telescopes